### PR TITLE
feat(ticket#246): Implementation of Attendance Breadcrumbs

### PR DESCRIPTION
**Pull request by nadirhussain786**
## Change Request

This PR introduces the Attendance Header, which includes a breadcrumb to display the titles of the following routes. Additionally, an "Create Attendance" button will appear when the route is `/attendance`. 

The header is implemented for the following routes:

- `/attendance`
-  `/attendance/add`
- `/attendance/edit/:id`
- `/attendance/mark`

### Resources

image

image

image

### CODE REVIEW

**Review by muhammad-bilal-ali-khokhar — APPROVED** [review]

File: src/app/(app)/attendance/add/page.tsx [hunk 1]
```diff
@@ -1,8 +1,19 @@
 import React from "react";
 import AttendanceFormBuilder from "../components/AttendanceFormBuilder";
+import { AttendanceHeader } from "../components/AttendanceHeader";
+import { Routes } from "@/lib/routes";
 
 const AttendanceForm = () => {
-  return <AttendanceFormBuilder />;
+  const breadcrumbs = [
+    { href: Routes.Attendance, label: "Attendance" },
+    { href: Routes.AddAttendance, label: "Add Attendance" },
```
Comment (Monfernape) [inline src/app/(app)/attendance/add/page.tsx:9]:
```suggestion
    { href: Routes.AddAttendance, label: "Add attendance" },
```

File: src/app/(app)/attendance/mark/page.tsx [hunk 2]
```diff
@@ -1,14 +1,21 @@
-import React from 'react'
```
Comment (Monfernape) [inline src/app/(app)/attendance/mark/page.tsx:1]: Wondering if we can rename this route to `/review`

**Review by Monfernape — APPROVED** [review]

File: src/app/(app)/attendance/add/page.tsx (hunk 1 above)
Comment (nadirhussain786) [reply src/app/(app)/attendance/add/page.tsx]: Fixed

File: src/app/(app)/attendance/mark/page.tsx (hunk 2 above)
Comment (nadirhussain786) [reply src/app/(app)/attendance/mark/page.tsx:1]: I have renamed this route

**Pull request merged**